infra/ansible/playbooks/node_exporter.yaml

```diff
@@ -0,0 +1,26 @@
+- name: Node Exporter Setup
+  hosts: "{{ host }}"
+
+  tasks:
```

Comment:
This Ansible task assumes the repo is already cloned, which might not be the case.

Also, there are some `sender` servers, like `aligned-holesky-stage-1-sender`, which @JuArce told me to test this PR on. Why are we running this ansible inside the Aggergator and Batcher?

Reply:
Yes, I made the playbook to be used after the repo was cloned intentionally, so we could include it within the playbooks of the Aggregator and Batcher. But I made a mistake when adding the playbook to the others, as the repository hadn't been cloned at that step yet 🤣. I'll fix that in a moment.

Reply:
Addressed in [#7923c1a](https://github.com/yetanotherco/aligned_layer/pull/1627/commits/7923c1aeb679737200d784d8d06f0bc2f4e94427)!